src/main/java/com/limechain/runtime/allocator/AllocationStats.java

```diff
@@ -0,0 +1,24 @@
+package com.limechain.runtime.allocator;
+
+import lombok.Data;
+
+import java.math.BigInteger;
+
+@Data
+public class AllocationStats {
+    private int bytesAllocated;
+    private int bytesAllocatedPeak;
+    private BigInteger bytesAllocatedSum;
```

Comment:
allocated:15 throws nullpointer exception
```suggestion
    private BigInteger bytesAllocatedSum = BigInteger.ZERO;
```